packages/simple-markdown/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@khanacademy/simple-markdown",
+  "version": "0.8.0",
+  "description": "Javascript markdown parsing, made simple",
+  "main": "simple-markdown.js",
+  "types": "simple-markdown.d.ts",
+  "scripts": {
+    "build": "make build",
+    "test": "make test",
+    "prepublishOnly": "make build",
+    "size": "size-limit"
+  },
+  "keywords": [
+    "markdown",
+    "simple",
+    "react",
+    "html",
+    "ast",
+    "markup",
+    "text processing",
+    "extension",
+    "flow",
+    "typescript"
+  ],
+  "author": "Khan Academy",
+  "license": "MIT",
+  "files": [
+    "index.js",
+    "simple-markdown.js",
+    "simple-markdown.d.ts",
+    "simple-markdown.min.js",
+    "README.md"
+  ],
+  "dependencies": {
+    "@types/react": ">=16.0.0"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/react-dom": ">=16.0.0",
+    "express": "^4.17.1",
+    "flow-bin": "^0.111.1",
+    "flow-typed": "^2.6.2",
+    "mocha": "^6.2.2",
+    "nyc": "^15.0.0",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "rollup": "^1.26.3",
+    "size-limit": "^0.21.1",
+    "typescript": "^3.6.4",
+    "uglify-js": "^3.6.7"
```

Comment:
I think most of these can be deleted.  This `package.json` only needs to define libraries that are unique dependencies for `simple-markdown` (the rest are specified in our root `package.json`)